### DISCUSSION TITLE
Support label position and help position

### DISCUSSION THREAD

**Discussion by boudewijn-zicht** [formkit/formkit#684] [Ideas]
Hello,

I would very much like to specify the position of the label (before, after, hidden) and position of the help text (before, after).

This is behaviour that is available in the FormKit predecessor vueformulate, but I can not find an equivalent in FormKit.

Any change that this is added, or do we have to solve this by override slots, i.e. the `wrapper` slot should be able to change the label position.

I look forward to your insights.

**Comment by justin-schroeder** [comment]
We found in VueFormulate while the label position prop was convenient it was also quite limiting as there were dozens of DOM elements and only that one had props to move it. It was unfeasible to add props for very single one especially as the inputs got more complicated. Instead, in formkit, we have the concept of the underlying schema. This allows you to re-arrange your inputs however you like without loosing functionality. You can do this by exporting inputs, or by using the sections-schema prop.

Exporting: https://formkit.com/guides/export-and-restructure-inputs
Sections-schema: https://formkit.com/essentials/inputs#sections-schema#sections-schema

**Reply by boudewijn-zicht** [reply]
Ahh, I see.  I will look into this schema approach.

Thanks for the quick response!
